app/src/main/java/com/khanhhuy/movie_db/adapter/CategoryAdapter.java

```diff
@@ -0,0 +1,57 @@
+package com.khanhhuy.movie_db.adapter;
+
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.khanhhuy.movie_db.BR;
+import com.khanhhuy.movie_db.R;
+import com.khanhhuy.movie_db.databinding.ItemListFragmentHomeBinding;
+import com.khanhhuy.movie_db.model.Category;
+
+import java.util.ArrayList;
+
+/**
+ * Created by yeu_thuong on 11/5/2017.
+ */
+
+public class CategoryAdapter extends RecyclerView.Adapter<CategoryAdapter.BindingHolder> {
+    private LayoutInflater mInflater;
+    private ArrayList<Category> mCategories;
+
+    public CategoryAdapter(ArrayList<Category> categories){
+        mCategories = categories;
+    }
+
+    @Override
+    public BindingHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = mInflater.from(parent.getContext()).inflate(R.layout.item_list_fragment_home, parent, false);
+        return new BindingHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(BindingHolder holder, int position) {
+        final Category category = mCategories.get(position);
+        holder.binding.setVariable(BR.category, category);
```

Comment:
define menthod bindData(Category category) in BindingHolder  and call 
holder.bindData is better design